maro/rl_v3/policy_trainer/distributed_discrete_maddpg.py

```diff
@@ -229,6 +229,43 @@ def get_batch_grad(
 
         return grad_dict
 
+    def _remote_learn(
```

Comment:
Shall we move `_remote_learn` and related logics to `AbsTrainWorker` since it is a general one used by all workers?